architecture/Threatmodel.md

```diff
@@ -0,0 +1,128 @@
+## 5. Overige app/ gebruiker interacties. 
+Dit is afhanklijk van de invulling van deze interacties. Te denken valt aan een test modus, extra informatie over Corona die getoond wordt aan de gebruiken.
+## Tegenstanders 
+### Script Kiddies. 
+Te denken valt aan jongeren die gebruiken maken van standaard programmatuur om zwakheden te ontdekken. Ze zijn meestal uit Nederland, vaak nog minderjarig. Een bekende klasse aanvallen die gebruikt worden zijn Denial of Service Attacks. Dit om de service te verstoren. Een voorbeeld hiervan zijn DDOS aanvallen op de internet betaal sites van de banken in 2018. 
+### Cyber criminelen 
+Dit zijn meestal in het buitenland operende criminelen die via computer hacks en afpersing geld proberen te verdienen. Het zijn de geprofessionaliseerde en gecriminaliseerde script kiddies. Ransomware aanvallen zijn de meest gebruikte gereedschappen voor dit soort criminelen. Vooral de centrale servers maar ook de code repository kunnen doelwit zijn van een afpersings zaak waar de  gegevens worden versleuteld of op een andere manier ontoegankelijk worden gemaakt, en alleen tegen betaling worden ontsleuteld.   
+### Buitenlandse mogendheden.
+Een contacten traceer app geeft ongekende mogelijkheden om mensen te volgen en om groepen (clusters) te ontdekken. In vele gevallen worden al mobiele telefoons in het buitenland bij grenspassage onderzocht. 
+### Corrupte medewerkers 
+Medewerkers kunnen rechten misbruiken en/ of informatie (bijvoorbeeld wachtwoorden) doorspelen naar derden en hierdoor gevoelige informatie lekken. Ook is het denkbaar dat bepaalde configuraties of code moedwillig wordt aangepast  om informatie te lekken.  
+### Politieke druk 
+Het huidige systeem is behoorlijk anoniem. Vanuit justitie (opsporing authoriteiten) kan er een druk verwacht worden om vooral in ernstige zaken toch de anonimiteit maatregelen op te heffen. Vergelijkbaar is bijvoorbeeld de discussie rond de whatsapp encryptie. Als dat eenmaal in een zaak is gedaan is de techniek van het doorbreken van de maatregelen bekend en ontstaat er een geleidende schaal van ernst van zaken, en zou het binnen enkele jaren gebruikt kunnen worden voor relatief lichte vergrijpen. Het is van groot belang voor de volksgezondheid dat een zo groot mogelijk aantal Nederlanders meedoet. Bij het gebruiken van contacten informatie voor andere doeleinden gaan de groepen die zich zorgen maken over de privacy de app niet meer gebruiken of gebruikmaken van tegen maatregelen zoals stoorzenders of elektromagnetische afscherming van ruimtes die ook vele andere nadelen hebben.  
+ ### Profiterende Programmeurs 
+De COVID-19 notificatie app is opensource. Andere programmeurs zouden in de verleiding kunnen komen om andere versies van de COVID-19 app te maken gebaseerd op de open source code met een aantal veranderingen. Hierdoor zou de privacy niet meer gewaarborgd worden of de AVG niet meer gevolgd bijvoorbeeld door een versie die advertenties laat zien. 
```

Comment:
Aangezien deze versies onder de EUPL (of compatibel) moeten worden gedistribueerd, staan er in principe exact dezelfde risico's voor deze alternatieve versies als de t.o.v. de originele programmeurs van de app, zoals onder het kopje corrupte medewerkers.

Reply:
De versie zou onder EUPL (of compatibel) moeten worden gedistribueerd dat klopt, maar om de licentie af te dwingen moet je de juridische weg bewandelen als men niet mee werkt.  Je kan natuurlijk ook aan Google/Apple vragen om de alternatieve app te verwijderen.